central/telemetry/centralclient/runtime_config.go

```diff
@@ -0,0 +1,81 @@
+package centralclient
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/pkg/env"
+	"github.com/stackrox/rox/pkg/telemetry/phonehome"
+)
+
+// getRuntimeConfig returns the runtime configuration with the configured key.
+// If returns nil or error, telemetry should be disabled.
+func getRuntimeConfig() (*phonehome.RuntimeConfig, error) {
+	if env.OfflineModeEnv.BooleanSetting() {
+		return nil, nil
+	}
+	runtimeCfg, err := phonehome.GetRuntimeConfig(
+		env.TelemetryConfigURL.Setting(),
+		env.TelemetryStorageKey.Setting(),
+	)
+	return runtimeCfg, errors.WithMessage(err, "failed to fetch runtime telemetry config")
+}
+
+func appendRuntimeCampaign(runtimeCfg *phonehome.RuntimeConfig) {
+	campaignMux.RLock()
```

Comment:
Shouldn't this be a write lock since we modify `telemetryCampaign`?

Reply:
Yes, it should, thanks!